src/features/broker-protection/comparisons/address.js

```diff
@@ -23,7 +23,7 @@ export function matchAddressFromAddressListCityState (userAddresses, foundAddres
             stateFound = possibleLocation.state
 
             if (isSameAddressCityState(userCity, userState, cityFound, stateFound)) {
-                return { cityFound, stateFound }
+                return true
             }
```

Comment:
I realised this method only ever considers a truthy result - so I made that explicit